pkg/controller/core.oam.dev/v1alpha2/application/application_controller.go

```diff
@@ -45,7 +45,7 @@ const RolloutReconcileWaitTime = time.Second * 3
 // Reconciler reconciles a Application object
 type Reconciler struct {
 	client.Client
-	dm     discoverymapper.DiscoveryMapper
+	Dm     discoverymapper.DiscoveryMapper
```

Comment:
where is this used outside of the package?

Reply:
Thanks for your review : ), Dm will be used in /test/compatibility-test/application/suit_test